packages/fast-components-react-msft/src/flipper/flipper.spec.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from "react";
+import * as Adapter from "enzyme-adapter-react-16";
+import { configure, shallow } from "enzyme";
+import examples from "./examples.data";
+import { generateSnapshots } from "@microsoft/fast-jest-snapshots-react";
+import {
+    FlipperProps,
+    IFlipperHandledProps,
+    IFlipperManagedClasses,
+    IFlipperUnhandledProps
+} from "./flipper";
+
+/*
+ * Configure Enzyme
+ */
+configure({adapter: new Adapter()});
+
+describe("flipper snapshots", (): void => {
+    generateSnapshots(examples);
+});
+
+describe("flipper", (): void => {
+    const Component: React.ComponentClass<IFlipperHandledProps> = examples.component;
+
+    test("should return an object that includes all valid props which are not enumerated as handledProps", () => {
+        const handledProps: IFlipperHandledProps = {
+            visible: false
+        };
+
+        const unhandledProps: IFlipperUnhandledProps = {
+            "aria-labelledby": "foo"
+        };
+
+        const props: IFlipperHandledProps & IFlipperUnhandledProps = {...handledProps, ...unhandledProps};
+
+        const rendered: any = shallow(
+            <Component {...props} />
+        );
+
+        const flipper: any = rendered.first().shallow();
+
+        expect(flipper.prop("aria-labelledby")).not.toBe(undefined);
```

Comment:
same here - the `.not.toBe(undefined` is covered with following test - the other one will fail if it is undefined